src/app/components/login/login.component.scss

```
@@ -0,0 +1,64 @@
+@import '/src/styles/colors';
+
+.container, .form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.user, .password, .login, .googleLogin {
+  border-radius: 15px;
+  border: 1px solid black;
+}
+
+.container {
+  width: 100%;
+  height: 100vh;
```

Ehenever do you usem `vh` measures always manage it with min-heaght or you could face layout problems in some devices.
```suggestion
  min-height: 100vh;
```